src/Tools/dotnet-watch/src/BrowserRefreshServer.cs

```diff
@@ -33,11 +33,13 @@ public BrowserRefreshServer(IReporter reporter)
 
         public async ValueTask<string> StartAsync(CancellationToken cancellationToken)
         {
+            var hostName = Environment.GetEnvironmentVariable("ASPNETCORE_AUTO_RELOAD_WS_HOSTNAME") ?? "127.0.0.1";
```

Comment:
@zbecknell the dotnet-watch arguments tend to use the DOTNET_WATCH prefix. Could you update it to use that instead?

```suggestion
            var hostName = Environment.GetEnvironmentVariable("DOTNET_WATCH_AUTO_RELOAD_WS_HOSTNAME") ?? "127.0.0.1";
```